app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb

```diff
@@ -1,11 +1,13 @@
-<p>Once you have a degree or equivalent qualification you’re ready for postgraduate primary or secondary initial teacher training courses.</p>
+<p>If you have a degree or equivalent qualification, you can do postgraduate primary or secondary teacher training.</p>
 
-<p>You also need the following GCSEs at grade 4 (C) or above, or equivalent qualifications:</p>
+<p>If you do not have a degree and are not studying for one, you can do <a href="/train-to-be-a-teacher/if-you-dont-have-a-degree">undergraduate teacher training</a>.</p>
```

Comment:
@EllieNodder Is this undergrad sentence in the right place? It is now in the middle of the postgrad info - is that confusing?

Reply:
My thinking was that the qualifications info was relevant for both postgrad and undergrad courses. And as a user, if I see 'if you have a degree', I'd then be thinking 'what about if I don't have a degree?' so thought it made sense to move that sentence further up. But equally happy to move it back down!

Reply:
If the qualifications section also applies to undergrad degrees then let's leave it as is